modules/Expectation.js

```diff
@@ -440,14 +440,35 @@ class Expectation {
     )
 
     assert(
-      spy.calls.some(call => isEqual(call.arguments, expectedArgs)),
+      spy.calls.some(call => call.arguments.some((argument, index) =>
+        argument === call.arguments[index]
+      )),
       'spy was never called with %s',
       expectedArgs
     )
 
     return this
   }
 
+  toNotHaveBeenCalledWith(...expectedArgs) {
+    const spy = this.actual
+
+    assert(
+      isSpy(spy),
+      'The "actual" argument in expect(actual).toNotHaveBeenCalledWith() must be a spy'
+    )
+
+    assert(
+      spy.calls.some(call => call.arguments.some((argument, index) =>
+        argument === call.arguments[index]
+      )),
+      'spy was not supposed to be called with %s',
+      expectedArgs
+    )
+
+    return this
+  }
+
```

Comment:
I created this just to be a little but familiarized with the codebase and the fact that the way the tests are structured I can't test of a `toHaveBeenCalledWith(args)` was not successful (it just acts as a positive assertion that can't be negated in the test itself so we need to rely in an opposite method to verify it).